### ISSUE TITLE
jobs: use 0xC player stats log line to determine gcd

### ISSUE THREAD

**Issue by quisquous** [quisquous/cactbot#595]
There's a couple of places where jobs has hardcoded gcd values.  It'd be nice to use the log line to know what your stats are and then calculate base gcd from there.

I'm not 100% sure of when this log line is sent, but if it's sent every time your stats change, this would be perfect.

**Comment by JLGarber** [comment]
It does seem to be sent on stat change. I went through a little and a few things that I confirmed DO send it:

- Entering a new instance/zone
- Eating food
- Changing jobs

I believe login also does, although that might be the same as zone changes. As for the line's components, I think this is the order, in case you don't already have it:

`[JOB NUMBER]|STR|DEX|VIT|INT|MND|PIE|ATTACK POWER|DHIT|CRIT|AMAG POTENCY|HMAG POTENCY|DET|SkS|SpS|???|TEN|???`

That first `???` was consistently 0 in the brief tests I ran. The second one looks like a hash, but I'm not sure which. (A sample is `941512565941b35ff495fe0fa984f327`.)

Aside, I'm genuinely puzzled by why Direct Hit and Crit are sandwiched like that.
